web-ui/app/templates/page/user_settings_box.hbs

```diff
@@ -0,0 +1,5 @@
+<header>
+  <h1 class="fa fa-user"> User Settings</h1>
+  <span id="user-settings-close"><i class="fa fa-close"></i></span>
+</header>
+<p>$account_email</p>
```

Comment:
Not sure if displaying the email like this will just work? I copied this from [here](https://github.com/pixelated/pixelated-user-agent/pull/502/files#diff-8a5e7bc9a61bb8cf18b2bd68bae60b94R9).